lib/renderer.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const marked = require('marked');
+const createDOMPurify = require('dompurify');
+const { JSDOM } = require('jsdom');
```

Comment:
Is it possible to only require `dompurify` when the option is enabled?

```suggestion
let JSDOM;
let createDOMPurify;

if (config.dompurify) {
  createDOMPurify = require('dompurify');
  JSDOM = require('jsdom').JSDOM;
}
```

This could speed up cold start performance.

Reply:
Ah, the suggestion here is just for demonstration. The `require` part should be added after the config is reader.

@tomap

Reply:
done